src/app/pages/swap/alex-swap-container.tsx

```diff
@@ -175,7 +175,7 @@ function AlexSwapContainer() {
       if (isSponsoredByAlex) {
         return await broadcastAlexSwap(txRaw);
       }
-      return await broadcastStacksSwap(unsignedTx);
+      return await broadcastStacksSwap(signedTx);
```

Comment:
this was the problem 😂😂 Not sure how I could've missed that.... 🤔

Reply:
At least I know understand the new signing code and pub/sub pattern. 🤣